src/ddsrt/src/process/windows/process.c

```diff
@@ -0,0 +1,386 @@
+/*
+ * Copyright(c) 2006 to 2018 ADLINK Technology Limited and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+ * v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+#include <errno.h>
+#include <assert.h>
+#include <process.h>
+#include "dds/ddsrt/heap.h"
+#include "dds/ddsrt/time.h"
+#include "dds/ddsrt/string.h"
+#include "dds/ddsrt/process.h"
+
+ddsrt_pid_t
+ddsrt_getpid(void)
+{
+  return GetCurrentProcessId();
+}
+
+
+#ifdef PROCESS_MANAGEMENT_ENABLED
+
+
+extern inline DWORD
```

Comment:
This is already done in `time.c` for Windows I think. Simply include the `dds/ddsrt/timeconv.h` header.